python/src/com/jetbrains/python/refactoring/move/makeFunctionTopLevel/PyBaseMakeFunctionTopLevelProcessor.java

```diff
@@ -89,7 +89,7 @@ protected final void performRefactoring(UsageInfo @NotNull [] usages) {
 
     assert ApplicationManager.getApplication().isWriteAccessAllowed();
 
```

Comment:
Same problem as with "Extract superclass". This processor is responsible for refactorings moving a method or a local function to the top-level of a file (triggered by the normal Move action). If the original function is inside a namespace package and is going to be moved to a different file, unnecessary `__init__.py` will be created. Please add tests on that (ideally both for "Make method top-level" and "Make local function top-level").